File: aoj/volume3/n0336/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int height = sc.nextInt();
        int radius = sc.nextInt();

        int sum = height + radius;

        if (sum == 0) {
            System.out.println(0);
        } else if (sum > 0) {
            System.out.println(1);
        } else if (sum < 0) {
            System.out.println(-1);
        }

    }

}

File: aoj/volume3/n0336/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int height, radius;
    cin >> height >> radius;

    int sum = height + radius;

    if (sum == 0) {
        cout << 0 << endl;
    } else if (sum > 0) {
        cout << 1 << endl;
    } else if (sum < 0) {
        cout << -1 << endl;
    }

    return 0;

}
